app/src/main/java/com/android/voyageur/ui/components/PlaceSearchWidget.kt

```diff
@@ -46,11 +62,22 @@ fun PlaceSearchWidget(
             DropdownMenuItem(
                 text = { Text(place.place.displayName ?: "Unknown Place") },
                 onClick = {
-                  onSelect(place)
+                  val location = place.toLocation()
+                  onSelect(location)
                   expanded = false
                 },
                 modifier = Modifier.testTag("item-${place.place.id}"))
           }
         }
   }
 }
+
+/** function to convert a CustomPlace object to a Location object with only the fields we need */
+fun CustomPlace.toLocation(): Location {
```

Comment:
This function should be in the CustomPlace class and used in the PlaceSearchWidget as this is an utility function that needs could be reused in other classes as well.